app/templates/main/search-transferring-body.html

```diff
@@ -22,6 +22,7 @@
     "Record status (closed)": "closure_type-asc",
     "Record status (open)": "closure_type-desc"
 } %}
+{% set anchor_fields = {"file_name": "file_id", "file_path": "file_id"} %}
```

Comment:
why is file_path in this anchor_fields dict?

Reply:
@anthonyhashemi removed it 🤝